src/main/scala/tech/cryptonomic/conseil/tezos/QueryProtocolTypes.scala

```diff
@@ -0,0 +1,36 @@
+package tech.cryptonomic.conseil.tezos
+
+import com.fasterxml.jackson.core.`type`.TypeReference
+import com.fasterxml.jackson.module.scala.JsonScalaEnumeration
+import tech.cryptonomic.conseil.tezos.QueryProtocolTypes.OperationType.OperationType
+
+object QueryProtocolTypes {
+  def mapOperationToSQL(operation: OperationType, inverse: Boolean): String = {
```

Comment:
what do you mean by `Slick operator`? I feel like I'm somehow bound to the plain SQL with Slick, as it does now want to work with more complex generic operations(it may look like it does sometimes in intellij, but sbt will tell you something different), I can move it

Reply:
I couldn't find a way to make it work with `Rep[A]`, that's why I'm sticking with string operations

Reply:
Moved it to `TezosDatabaseOperations`, I think it fits there as a private method